packages/node_modules/@node-red/nodes/core/parsers/70-CSV.js

```diff
@@ -231,16 +231,16 @@ module.exports = function(RED) {
                             }
                         }
             			else {
```

Comment:
if we are being picky - then this doesn't line up either :-)

Reply:
Thanks! I try to fix also line 233.